profiling/build.rs

```diff
@@ -6,6 +6,9 @@ use std::path::PathBuf;
 use std::process::Command;
 
 fn main() {
+    // This makes sure `build.rs` will be run on every `cargo build` command and helps when
+    // switching PHP versions in local dev environment and when building in CI
+    println!("cargo:rerun-if-changed=NULL");
```

Comment:
I need to remove this again. This slows down local rebuilds (and the local rust-analyzer that I use as LSP) heavily. Sadly I got lots of "interesting" build errors in CI if I remove it, even a `cargo clean` would not solve these, so clearly this is not a solution to the problem.